docs/tutorials/kafka.ipynb

```diff
@@ -0,0 +1,1207 @@
+{
```

Comment:
<p>Is the accuracy low and not improving per epoch because of the small fraction of training data used? You might want to mention that and run less epochs if that's the case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
added the comments in the commit